pkg/skaffold/docker/image_util.go

```diff
@@ -56,3 +59,29 @@ func Prune(ctx context.Context, out io.Writer, images []string, client LocalDaem
 	}
 	return nil
 }
+
+func RetrieveWorkingDir(tagged string, insecureRegistries map[string]bool) (string, error) {
+	var cf *v1.ConfigFile
+	var err error
+
+	if strings.ToLower(tagged) == "scratch" {
+		return "/", nil
+	}
+
+	localDocker, err := NewAPIClient(false, nil)
+	if err != nil {
+		// No local Docker is available
+		cf, err = RetrieveRemoteConfig(tagged, insecureRegistries)
+	} else {
+		cf, err = localDocker.ConfigFile(context.Background(), tagged)
```

Comment:
I realized after making this comment you didn't write this code, but going to leave it here in case anyone has opinions on it:

do we not want to try and retrieve the image config from remote if we were able to get a local docker client, but couldn't find the image in the local daemon?

Reply:
good catch. Please have another look.